src/saved_model.rs

```diff
@@ -124,6 +124,16 @@ impl TensorInfo {
     pub fn name(&self) -> &OutputName {
         &self.name
     }
+    
```

Comment:
Please remove the whitespace on these blank lines.

Reply:
Done.